packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -552,6 +572,7 @@ export class List<T = any> extends BaseComponent<IListProps<T>, IListState<T>> i
    * This function is debounced.
    */
   private _onScrollingDone(): void {
+    this._onAsyncIdle(); // Guarantees the creation of lookahead images when scrolling is done
```

Comment:
This may cause perf regressions, we'll need to look at the profiler. `_onAsyncIdle` is expensive IIRC.

Reply:
I did some perf tests and it seems OK to remove this.  I am brining over some changes made by other developers, so I am not sure why this was necessary.  I have removed it.  Thank you.